test/cmocka/src/lib/alloc/alloc.c

```diff
@@ -298,6 +306,26 @@ static void test_lib_alloc_immediate_free(struct test_case *tc)
 	}
 }
 
+static void test_lib_rballoc_filling_preceding_heaps(struct test_case *tc)
+{
+	size_t first_heap_size = HEAP_BUFFER_SIZE - tc->alloc_size * 8;
+	size_t second_heap_size = HEAP_HP_BUFFER_SIZE - tc->alloc_size;
+	size_t third_heap_size = 2048;
+	int *first_mem = rballoc(tc->alloc_zone, SOF_MEM_CAPS_RAM,
+				 first_heap_size);
+	int *second_mem = rballoc(tc->alloc_zone, SOF_MEM_CAPS_RAM,
+				  second_heap_size);
+	int *third_mem = rballoc(tc->alloc_zone, SOF_MEM_CAPS_RAM,
+				 third_heap_size);
+
+	assert_ptr_equal(first_mem, HEAP_BUFFER_BASE);
+	assert_ptr_equal(second_mem, HEAP_HP_BUFFER_BASE);
+	assert_ptr_equal(third_mem, HEAP_LP_BUFFER_BASE);
+	rfree(first_mem);
```

Comment:
It would be also good to have empty line here. It's good to visually split things that are related to different things.

Reply:
@dbogdzix Thanks